packages/edit-site/src/components/navigation-sidebar/navigation-panel/content-navigation-item.js

```diff
@@ -76,6 +76,7 @@ export default function ContentNavigationItem( { item } ) {
 			{ isPreviewVisible && previewContent && (
 				<NavigationPanelPreviewFill>
 					<TemplatePreview
+						className="edit-site-navigation-panel__preview"
 						rawContent={ previewContent }
 						blockContext={ {
```

Comment:
Just noting this doesn't seem to work, although it doesn't on `trunk` either if I am interpereting this correctly.

This would be the preview from hovering a content item in the nav panel?  Such as Content: Pages -> "Sample Page" etc?  Nothing appears for me like it does for the template-navigation-items - although thats not a result of this PR. 🤔

Reply:
I think this code is for the template navigation items preview. I never saw previews for individual pages and posts, I'm not sure we should have them or not. But given that the behavior is the same in the trunk I think we can try to address that on a different PR.

Reply:
>But given that the behavior is the same in the trunk I think we can try to address that on a different PR.

Agreed!  Il resolve this conversation and open an issue regarding that.

>I think this code is for the template navigation items preview. I never saw previews for individual pages and posts

The previews that handle the template navigation items are under the `template-navigation-item.js` sibling file.  I don't remember seeing previews for the content section before either, but looking closer this seems to be what that is supposed to do.  As these are the items that get shown under Content: Pages > / Posts > / Categories >.

Reply:
Seems its an easy fix: https://github.com/WordPress/gutenberg/pull/34203